### PR TITLE
replace django.core.urlresolvers with django.urls

### PR DESCRIPTION
**Pull request by nickpell**
@esoergel

### CODE REVIEW

**Comment by esoergel** [comment]
```
ImportError: No module named 'django.urls'
```
Guessing you need to update some test requirements or something.

File: setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     packages=find_packages(),
     zip_safe=False,
     install_requires=[
-        'django>=1.7a,<1.9',
+        'django>=1.10,<1.11',
```
Comment (millerdev) [inline setup.py]: I know we've moved to Django 1.10, but wondering if it's not possible to still support earlier versions of Django? I didn't look at how the url APIs have changed, so maybe it just makes sense to only support 1.10 for some reason?

Comment (esoergel) [inline setup.py]: I'm not sure what it'd take to be backwards compatible, but I do think it'd be pretty meaningless to say we support (for example) 1.7 unless we at least run the test suite against it regularly.

Comment (millerdev) [inline setup.py]: good point, and I'm fine with `>=1.10` unless it's very easy to expand the test matrix to run on more versions of Django

Comment (nickpell) [inline setup.py]: trying https://docs.travis-ci.com/user/languages/python/#Testing-Against-Multiple-Versions-of-Dependencies-(e.g.-Django-or-Flask)

File: .travis.yml [hunk 2]
```diff
@@ -3,11 +3,11 @@ python:
...
 env:
- - DJANGO_VERSION=1.8.17
- - DJANGO_VERSION=1.9.12
- - DJANGO_VERSION=1.10.6
+ - DJANGO_VERSION_MIN=1.8 DJANGO_VERSION_MAX=1.9
+ - DJANGO_VERSION_MIN=1.9 DJANGO_VERSION_MAX=1.10
+ - DJANGO_VERSION_MIN=1.10 DJANGO_VERSION_MAX=1.11
 install:
- - "pip install django==$DJANGO_VERSION"
+ - "pip install django=>$DJANGO_VERSION_MIN,<$DJANGO_VERSION_MAX"
```
Comment (millerdev) [inline .travis.yml]: I didn't know `=>` was a valid operator. Should that be: `...django>=$DJANGO...`

Comment (nickpell) [inline .travis.yml]: whoops

**Comment by nickpell** [comment]
figured out the test matrix.  now need to figure out why tests are failing with django>=1.9

**Comment by nickpell** [comment]
✅

**Pull request merged**